lib/gooddata/rest/connection.rb

```diff
@@ -617,37 +574,26 @@ def scrub_params(params, keys)
         new_params
       end
 
-      def update_stats(title, delta)
-        synchronize do
-          orig_title = title
-
-          placeholders = true
-
-          if placeholders
-            title = self.class.map_placeholders(title)
-          end
-
-          stat = stats[title]
-          if stat.nil?
-            stat = {
-              :min => delta,
-              :max => delta,
-              :total => 0,
-              :avg => 0,
-              :calls => 0,
-              :entries => []
-            }
-          end
-
-          stat[:min] = delta if delta < stat[:min]
-          stat[:max] = delta if delta > stat[:max]
-          stat[:total] += delta
-          stat[:calls] += 1
-          stat[:avg] = stat[:total] / stat[:calls]
+      def anonymize_path(path)
```

Comment:
info: Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction) (reek)